From cf5541ac88bfeb4c8275f71c0f8f94cdcd7bf8cb Mon Sep 17 00:00:00 2001
From: John Kirkham <kirkhamj@janelia.hhmi.org>
Date: Fri, 3 Nov 2017 11:39:18 -0400
Subject: [PATCH] Import os in tests

Later in the file `os.listdir` is used as part of the macOS tests, but
is not being imported. This fixes that.
---
 constructor/tests/__init__.py | 1 +
 1 file changed, 1 insertion(+)

diff --git constructor/tests/__init__.py constructor/tests/__init__.py
index e99a1f3..0056e7c 100644
--- constructor/tests/__init__.py
+++ constructor/tests/__init__.py
@@ -6,6 +6,7 @@
 
 from __future__ import absolute_import, division, print_function
 
+import os
 from os.path import dirname
 import sys
 
